autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/di/AutofillModule.kt

```diff
@@ -43,6 +44,6 @@ class AutofillModule {
         context: Context,
         internalTestUserChecker: InternalTestUserChecker
     ): AutofillStore {
-        return SecureStoreBackedAutofillStore(secureStorage, context, internalTestUserChecker)
+        return SecureStoreBackedAutofillStore(secureStorage, context, internalTestUserChecker, RealLastUpdatedTimeProvider())
```

Comment:
Should we declare a `@Provides` or `@ContributesBinding` for the time provider, instead of instantiating here? 

functionally, not going to matter too much, but more in line with how we do it elsewhere

Reply:
I added RealLastUpdatedTimeProvider / LastUpdatedTimeProvider in autofill-store so we don't have DI there that's why I chose to do it this way.